.travis.yml

```diff
@@ -41,21 +41,19 @@ install:
   - conda update conda
 
   # Install dependencies
-  - conda create -n odo python=$TRAVIS_PYTHON_VERSION pytest numpy pandas sqlalchemy
-  - conda install -n odo h5py pip cython bcolz coverage networkx toolz multipledispatch
-  - conda install -n odo pytables paramiko boto boto3 bokeh pymysql
-  - if [ -n "$PANDAS_VERSION" ]; then conda remove -n odo pandas; fi
+  # Use conda **ONLY** for numpy and pandas, this speeds up the builds a
+  # lot. Use the normal pip install for the rest.
+  - conda create -n odo python=$TRAVIS_PYTHON_VERSION pandas=${PANDAS_VERSON:-0.19.0} numpy=1.11.2
```

Comment:
I like that users can just pip install and go on their local systems, what do you think about putting a comment in the reqs file next to pandas and numpy to tell users to update the .travis file?